test/tests/postprocessors/side_integral/tests

```diff
@@ -15,8 +16,46 @@
 
     issues = '#1405 #16099'
     design = 'SideIntegralVariablePostprocessor.md'
-    requirement = 'The system shall support computing the integral of a finite volume variable'
-        'quantity over a side on the mesh.'
+    requirement = 'The system shall support computing the integral of a finite volume variable '
+                  'quantity over a side on the mesh.'
     ad_indexing_type = 'global'
   []
+
+  [side_integral_material_property]
+    issues = '#18649'
+    design = 'SideIntegralMaterialProperty.md'
+    requirement = 'The system shall support computing the integral of a material property over a '
+                  'side on the mesh.'
+    [real]
+      type = 'CSVDiff'
+      input = '../side_material.i side_integral_material_property.i'
+      csvdiff = 'side_integral_material_property_out.csv'
+      cli_args = 'Materials/active=real'
+      detail = 'if the material property is of `Real` type'
+    []
+    [realvector]
+      type = 'CSVDiff'
+      input = '../side_material.i side_integral_material_property.i'
+      csvdiff = 'side_integral_material_property_out.csv'
+      cli_args = 'Materials/active=realvector Postprocessors/integral/component="1"'
+      detail = 'if the material property is of `RealVectorValue` type'
```

Comment:
we usually have the requirements try to be code agnostic, so like 'of vector type' or 'for vector material properties'